### ISSUE TITLE
addon currently broken

### ISSUE THREAD

**Issue by invakid404** [FlowingCode/XTermConsoleAddon#1]
When trying to use this addon, a client-side error occurs:
```
Uncaught TypeError: Cannot read property 'insertMode' of undefined
    at t.<anonymous> (vaadin-bundle-99d7ed33dfb94b6a56a7.cache.js:6958)
    at i.parse (vaadin-bundle-99d7ed33dfb94b6a56a7.cache.js:2380)
    at t.parse (vaadin-bundle-99d7ed33dfb94b6a56a7.cache.js:2380)
    at e._action (vaadin-bundle-99d7ed33dfb94b6a56a7.cache.js:2380)
    at e._innerWrite (vaadin-bundle-99d7ed33dfb94b6a56a7.cache.js:2380)
    at vaadin-bundle-99d7ed33dfb94b6a56a7.cache.js:2380
```
Even the demo itself doesn't seem to work due to the same error.

**Comment by javier-godoy** [comment]
Hello. Thanks for the bug report. There were some breaking changes in xterm 4.4.0 and 4.5.0, and this addon was using a range dependency.
For now, it's fixed by pinning npm dependencies to xterm 4.3.0 and xterm-addon-fit 0.3.0. 

I've just deployed the demo site, I'll release a new version of this addon soon. (If you have built it locally, you may need to remove pacakge-lock.json and do clean build, otherwise it might use the alredy downloaded incompatible version of xterm)

Fixed in XTermConsoleAddon 1.0.1

**Comment by javier-godoy** [comment]
The release will be uploaded to Vaadin directory on Monday. 
Meanwhile, you can use the jar from the following repository:

```
<repository>
   <id>flowingcode-addons</id>
   <url>https://maven.flowingcode.com/addons/com/flowingcode/addons</url>
</repository>
```
